ui/src/pages/custody/Account.tsx

```diff
@@ -83,111 +54,55 @@ const Account: React.FC<ServicePageProps<Service> & AccountProps> = ({
-    if (!service)
-      return displayErrorMessage({
-        message: 'The account provider does not offer issuance services.',
-      });
-    await ledger.exercise(Service.RequestCloseAccount, service.contractId, {
-      accountId: c.payload.account.id,
+    await ledger.exercise(Service.RequestWithdrawal, service.contractId, {
+      depositCid: c.contractId,
     });
   };
 
   return (
     <>
-      <InputDialog {...transferDialogProps} isModal />
       <div className="account">
         <div className="account-details">
           <div className="account-data">
             <h4> {targetAccount.account.id.label} </h4>
-            {normalAccount &&
-              targetAccount.account.owner === party &&
-              (existingCloseRequest ? (
-                <p className="close-request">
-                  <i> close request pending</i>
-                </p>
-              ) : (
-                <OverflowMenu>
-                  <OverflowMenuEntry
-                    label={'Close Account'}
-                    onClick={() => requestCloseAccount(normalAccount)}
-                  />
-                </OverflowMenu>
-              ))}
           </div>
           <div className="account-data body">
-            <p className="p2">Type: {normalAccount ? 'Normal' : 'Allocation'} </p>
             <p className="p2">Provider: {getName(targetAccount.account.provider)}</p>
             <p className="p2">Owner: {getName(targetAccount.account.owner)}</p>
             <p className="p2">
               Role: {party === targetAccount.account.provider ? 'Provider' : 'Client'}
             </p>
-            {allocationAccount && (
-              <p className="p2">Nominee: {allocationAccount.payload.nominee}</p>
-            )}
-            {normalAccount && (
-              <p className="p2">
-                Controllers:{' '}
-                {damlSetValues(normalAccount.payload.ctrls)
-                  .map(ctrl => getName(ctrl))
-                  .sort()
-                  .join(', ')}
-              </p>
-            )}
+            <p className="p2">
+              Signatories:{' '}
+              {damlSetValues(targetAccount.account.id.signatories)
+                .map(a => getName(a))
+                .sort()
+                .join(', ')}
+            </p>
           </div>
         </div>
         {accountDeposits.length > 0 ? (
           accountDeposits.map(c => (
             <Tile className="account-holding" key={c.contractId}>
               <p>
                 <b>{c.payload.asset.id.label}</b> {c.payload.asset.quantity}{' '}
+                {!isEmptySet(c.payload.lockers) && (
+                  <Popup
+                    trigger={<Icon name="lock" />}
+                    content={
+                      'Locked by ' +
+                      damlSetValues(c.payload.lockers)
+                        .map(a => getName(a))
+                        .sort()
+                        .join(', ')
+                    }
```

Comment:
`<ToolTip/>` could also work here!

Reply:
Not sure what you're referring to here with `<ToolTip/>` as I cannot find it in the project. I only see `<div className="tool-tip">Click here to fit to screen</div>` which is on the ReviewPage. Are you referring to this ?

Reply:
Nope. My mistake - when I suggested this I searched for the term in the project and got a false positive thinking that _that_ instance of ToolTip was a common one - never mind!